README.md

```diff
@@ -45,7 +45,7 @@ Usage: cli [options] <asyncapi> <template>
 
 Options:
   -V, --version                  output the version number
-  -d, --disable-hook <hookName>  disable a specific hook
+  -d, --disable-hook <hookName>  disable a specific hook type
```

Comment:
`<hookType>` or `<hookTypeName>` instead of `<hookName>`?

Reply:
will do `<hookType>`